$OpenBSD: patch-conserver_main_c,v 1.3 2019/02/25 19:53:16 sthen Exp $

- enable "setproctitle" option by default, for more information ps listings
- libressl doesn't support @SECLEVEL=0

Index: conserver/main.c
--- conserver/main.c.orig
+++ conserver/main.c
@@ -70,7 +70,7 @@ CONFIG defConfig =
     DEFPORT,
     FLAGTRUE, FLAGTRUE, 0, DEFBASEPORT, (char *)0, 0
 #if HAVE_SETPROCTITLE
-	, FLAGFALSE
+	, FLAGTRUE
 #endif
 #if HAVE_OPENSSL
 	, (char *)0, FLAGTRUE, FLAGFALSE, (char *)0
@@ -363,7 +363,7 @@ SetupSSL(void)
 	    }
 	    ciphers = "ALL:!LOW:!EXP:!MD5:!aNULL:@STRENGTH";
 	} else {
-	    ciphers = "ALL:aNULL:!LOW:!EXP:!MD5:@STRENGTH" CIPHER_SEC0;
+	    ciphers = "ALL:aNULL:!LOW:!EXP:!MD5:@STRENGTH";
 	}
 	if (config->sslcacertificatefile != (char *)0) {
 	    STACK_OF(X509_NAME) * cert_names;
